### PR TITLE
NAMECHEAP: New maintainer: willpower232

### PR DESCRIPTION
**Pull request by willpower232**
As per emails with @tlimoncelli, I should like to volunteer to maintain the namecheap provider.

<details>
<summary>
I have been able to run the tests against the namecheap sandbox and they almost all pass.
</summary>

```
--- FAIL: TestDNSProviders (197.07s)
    --- FAIL: TestDNSProviders/willpower232testsdnscontrol.com (197.07s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Clean_Slate:Empty (1.36s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/00:A:Create_A (1.17s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/00:A:Change_A_target (1.02s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty (0.75s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/01:Attl:Create_Arc (1.04s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/01:Attl:Change_TTL (1.15s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#01 (0.75s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/02:MX:Create_MX (1.05s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/02:MX:Change_MX_target (1.04s)
        --- FAIL: TestDNSProviders/willpower232testsdnscontrol.com/02:MX:Change_MX_p (1.11s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#02 (0.73s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/03:CNAME:Create_a_CNAME (1.05s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#03 (0.61s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/04:ManyAtOne:CreateManyAtLabel (0.93s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#04 (0.69s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/05:manyAtOneTypes:CreateManyTypesAtLabel (1.01s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#05 (0.83s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/06:Apex:Create_A (1.16s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#06 (0.64s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/07:TTL:Start (47.60s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#07 (0.59s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/08:add_to_label_and_change_orig_ttl:Setup (0.93s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#08 (0.71s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/09:Protocol-Wildcard:Create_wildcard (0.92s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#09 (0.64s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/10:TypeChange:Create_a_CNAME (1.04s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#10 (0.67s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/11:CommonDNS:Create_1_of_each (0.93s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#11 (0.72s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/12:CNAME:Record_pointing_to_@ (1.08s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#12 (0.61s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/13:MX:Record_pointing_to_@ (1.49s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#13 (0.63s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/14:NS:NS_for_subdomain (0.93s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#14 (0.77s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/15:complex_TXT:TXT_with_0-octel_string (0.95s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#15 (0.71s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/16:Case_Sensitivity:Create_CAPS (1.07s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#16 (0.62s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/17:IDNA:Internationalized_name (47.63s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#17 (1.18s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/18:IDNAs_in_CNAME_targets:IDN_CNAME_AND_Target (1.03s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#18 (0.78s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/19:pager101:99_records (1.64s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#19 (0.73s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/20:pager601_***SKIPPED(disabled_by_only)***:Empty (0.68s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/21:pager1201_***SKIPPED(disabled_by_only)***:Empty (0.60s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/22:NS1_URLFWD_tests_***SKIPPED(disabled_by_only)***:Empty (0.27s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/23:CAA:CAA_record (0.86s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#20 (1.24s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/24:NAPTR_***SKIPPED(CanUseNAPTR_not_supported)***:Empty (0.28s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/25:PTR_***SKIPPED(CanUsePTR_not_supported)***:Empty (0.36s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/26:SOA_***SKIPPED(CanUseSOA_not_supported)***:Empty (0.26s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/27:SRV_***SKIPPED(CanUseSRV_not_supported)***:Empty (0.27s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/28:SSHFP_***SKIPPED(CanUseSSHFP_not_supported)***:Empty (0.58s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/29:TLSA_***SKIPPED(CanUseTLSA_not_supported)***:Empty (0.28s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/30:DS_***SKIPPED(CanUseDS_not_supported)***:Empty (0.32s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/31:DS_(children_only)_***SKIPPED(CanUseDSForChildren_not_supported)***:Empty (0.28s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/32:DS_(children_only)_CLOUDNS_***SKIPPED(CanUseDSForChildren_not_supported)***:Empty (0.28s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/33:ALIAS:ALIAS_at_root (0.88s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#21 (0.78s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/34:AZURE_ALIAS_A_***SKIPPED(CanUseAzureAlias_not_supported)***:Empty (0.27s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/35:AZURE_ALIAS_CNAME_***SKIPPED(CanUseAzureAlias_not_supported)***:Empty (0.27s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/36:R53_ALIAS2_***SKIPPED(CanUseRoute53Alias_not_supported)***:Empty (0.29s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/37:R53_ALIAS_ORDER_***SKIPPED(CanUseRoute53Alias_not_supported)***:Empty (0.27s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/38:R53_ALIAS_CNAME_***SKIPPED(CanUseRoute53Alias_not_supported)***:Empty (0.29s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/39:CF_REDIRECT_***SKIPPED(disabled_by_only)***:Empty (0.27s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/40:CF_PROXY_***SKIPPED(disabled_by_only)***:Empty (0.27s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/41:CF_WORKER_ROUTE_***SKIPPED(disabled_by_only)***:Empty (0.53s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/42:IGNORE_NAME_function:Create_some_records (0.88s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#22 (0.68s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/43:IGNORE_NAME_apex:Create_some_records (47.52s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#23 (0.61s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/44:IGNORE_TARGET_function:Create_some_records (0.92s)
        --- PASS: TestDNSProviders/willpower232testsdnscontrol.com/Post_cleanup:Empty#24 (1.03s)
=== RUN   TestDualProviders
    integration_test.go:320: Skipping.  DocDualHost == Cannot
--- SKIP: TestDualProviders (0.00s)
FAIL
exit status 1
FAIL	github.com/StackExchange/dnscontrol/v3/integrationTest	197.080s
```
</details>

### CODE REVIEW

**Comment by cafferata** [comment]
Can you also:
1. Remove the warning from the [Namecheap provider page](https://docs.dnscontrol.org/service-providers/providers/namecheap)?
2. Add your self to the list of maintainers in the [documentation/providers list](https://docs.dnscontrol.org/service-providers/providers#:~:text=%40koesie10-,NAMECHEAP,-VOLUNTEER%20NEEDED).

File: README.md [hunk 1]
```diff
@@ -147,9 +147,9 @@ DNSControl can be installed via packages for macOS, Linux and Windows, or from s
 
 See [dnscontrol-action](https://github.com/koenrh/dnscontrol-action) or [gacts/install-dnscontrol](https://github.com/gacts/install-dnscontrol).
 
-## Deprecation warnings (updated 2023-01-18)
+## Deprecation warnings (updated 2023-02-18)
 
-- **Call for new volunteer maintainers for NAMECHEAP, NAMEDOTCOM, and SOFTLAYER.** These providers have no maintainer. Maintainers respond to PRs and fix bugs in a timely manner, and try to stay on top of protocol changes.
+- **Call for new volunteer maintainers for NAMEDOTCOM, and SOFTLAYER.** These providers have no maintainer. Maintainers respond to PRs and fix bugs in a timely manner, and try to stay on top of protocol changes.
```
Comment (cafferata) [inline README.md:152]:
```suggestion
- **Call for new volunteer maintainers for NAMEDOTCOM and SOFTLAYER.** These providers have no maintainer. Maintainers respond to PRs and fix bugs in a timely manner, and try to stay on top of protocol changes.
```

**Review by cafferata — APPROVED** [review]

**Comment by cafferata** [comment]
Thanks for handling the feedback. 👍

**Comment by willpower232** [comment]
Not a problem! 🙏

**Pull request merged**